charts/jaeger/values.yaml

```diff
@@ -187,15 +187,25 @@ storage:
 
 # Begin: Override values on the Cassandra subchart to customize for Jaeger
 cassandra:
+  replicaCount: 1
+
+  resources:
+    type: "small"
+
   persistence:
     # To enable persistence, please see the documentation for the Cassandra chart
     enabled: false
-  config:
-    cluster_name: jaeger
-    seed_size: 1
-    dc_name: dc1
-    rack_name: rack1
-    endpoint_snitch: GossipingPropertyFileSnitch
+
+  cluster:
+    name: jaeger
+    datacentre: dc1
```

Comment:
"centre", British spelling - is this correct?

Reply:
I'm on my phone and can't check now but this should be in the tests somewhere. This value should be in the output yaml.